documentation/modules/overview/con-feature-gates-overview.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-overview-components.adoc
+
+[id='con-feature-gates-overview-{context}']
+= Feature gates in Strimzi Operators
+
+You can enable and disable some features of Operators using _feature gates_. 
```

Comment:
We're using lower-case _operator_ in `ref-operator-cluster.adoc`

Reply:
Changed to lower-case to match.